AK/Bitmap.h

```diff
@@ -38,7 +38,7 @@ class Bitmap {
     }
 
     [[nodiscard]] BitmapView view() { return { m_data, m_size }; }
-    [[nodiscard]] BitmapView const view() const { return { m_data, m_size }; }
+    [[nodiscard]] BitmapView view() const { return { m_data, m_size }; }
```

Comment:
Out of scope, but this sounds like it really should be `const`. Also, why is `BitmapView::set()` marked `const`? This subverts the entire point of `const`!

Reply:
The return object is copied to the client and is copyable, so making it const is not a useful thing to do. You'd need a ConstBitmapView to do what you're saying.

Anyway, this const did nothing.

Reply:
I agree that it did nothing, and I agree that implementing a ConstBitmapView is out of scope :)